assets/js/pages/UpgradablePackagesPage/UpgradablePackagesPage.jsx

```diff
@@ -11,9 +11,8 @@ import {
   fetchUpgradablePackagesPatches,
 } from '@state/softwareUpdates';
 
-import PageHeader from '@common/PageHeader';
 import BackButton from '@common/BackButton';
-import UpgradablePackagesList from '@common/UpgradablePackagesList';
+import UpgradablePackages from './UpgradablePackages';
```

Comment:
It seems like this was a missed rename?

Reply:
No, it is a separate component. I split the navigation and redux into a seperate thing